php/libraries/Data/DTO.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace LORIS\Data;
+
+abstract class DTO
```

Comment:
I'm not sure how you're distinguishing DTO and DAO or what you intend the difference to be.

Is a DTO just a serializable DAO?

Reply:
DAO should take care of Selecting/Updating/Deleting rows in the database. The only way to interact with a DAO should be by using a Data Transfer Object (DTO). DTO should have getters and setters plus very limited logic e.g. __toString and the like)